src/schema/v2/ordered_set.ts

```diff
@@ -36,6 +38,24 @@ const OrderedSetType = new GraphQLObjectType<any, ResolverContext>({
         })
       },
     },
+    itemsConnection: {
```

Comment:
We're using an `OrderedSet` to store the list of artworks that are "featured" in the grid on the art keeps going page, but the `ArtworkGrid` needs a connection.

Eventually this could be extended to be a union type similar to `items`. For not it explicitly only resolves for artworks.